ctru-rs/src/services/am.rs

```diff
@@ -29,6 +33,14 @@ impl<'a> Title<'a> {
         self.id
     }
 
+    pub fn low_u32(&self) -> u32 {
+        (self.id & 0x0000_0000_ffff_ffff) as u32
+    }
+
+    pub fn high_u32(&self) -> u32 {
+        ((self.id & 0xffff_ffff_0000_0000) >> 32) as u32
+    }
```

Comment:
It isn't clear what these functions are doing from the name & signature. Maybe they can be inlined. If these functions are really needed, they should probably be private and well documented.